taxcalc/params.json

```diff
@@ -920,9 +920,9 @@
     },
     "_NIIT_trt":{
         "long_name": "Net Investment Income Tax rate (UIMC)",
-	    "description": "AKA Unearned Income Medicare Distribution Tax. Taxed on net investment income of individuals with MAGI higher than certain thresholds.",
-        "irs_ref": "Form , line , ",
-        "notes": "HC",
+	"description": "Also called Unearned Income Medicare Contribution Tax. You will be taxed at this rate if your MAGI is higher than NIIT threshold.",
```

Comment:
On Tue, 4 Aug 2015, Amy Xu wrote:

> 
> In taxcalc/params.json:
> 
> > @@ -920,9 +920,9 @@
> >      },
> >      "_NIIT_trt":{
> >          "long_name": "Net Investment Income Tax rate (UIMC)",
> > -	    "description": "AKA Unearned Income Medicare Distribution Tax. Taxed 
> on net investment income of individuals with MAGI higher than certain thresh
> olds.",
> > -        "irs_ref": "Form , line , ",
> > -        "notes": "HC",
> > +	"description": "Also called Unearned Income Medicare Contribution Tax. Yo
> u will be taxed at this rate if your MAGI is higher than NIIT threshold.",
> 
> If I remember correctly, the tax rate is applied to the entire investment
> income rather than the portion above the threshold, I'll check again.

See paragraph 20 at:

   http://www.irs.gov/uac/Newsroom/Net-Investment-Income-Tax-FAQs

dan

> 
> —
> Reply to this email directly or view it on
> GitHub.[AHvQVWjxLuRTFqFPRcctZ66BHb99KsZsks5okLYSgaJpZM4FhcK0.gif]
> 
> 
>